jpa/subsystem/src/main/java/org/jboss/as/jpa/management/EntityManagerFactoryLookup.java

```diff
@@ -40,7 +39,7 @@
     public EntityManagerFactory entityManagerFactory(final String scopedPersistenceUnitName) {
         PersistenceUnitService persistenceUnitService = PersistenceUnitRegistryImpl.INSTANCE.getPersistenceUnitService(scopedPersistenceUnitName);
         if (persistenceUnitService == null) {
-            throw JpaLogger.ROOT_LOGGER.PersistenceUnitNotAvailable(scopedPersistenceUnitName);
+            return null;
```

Comment:
I think that this is the only caller to paLogger.ROOT_LOGGER.PersistenceUnitNotAvailable(scopedPersistenceUnitName), so should comment out the PersistenceUnitNotAvailable() so it doesn't need to be translated.